package.json

```diff
@@ -6,71 +6,43 @@
   "dependencies": {
     "@data-driven-forms/pf4-component-mapper": "^3.22.3",
     "@data-driven-forms/react-form-renderer": "^3.22.3",
-    "@patternfly/react-core": "^5.2.0",
-    "@patternfly/react-icons": "^5.2.0",
-    "@patternfly/react-table": "^5.2.0",
+    "@patternfly/react-core": "^5.4.1",
+    "@patternfly/react-icons": "^5.4.0",
+    "@patternfly/react-table": "^5.4.0",
     "@redhat-cloud-services/frontend-components": "^4.2.8",
     "@redhat-cloud-services/frontend-components-notifications": "^4.1.0",
-    "@redhat-cloud-services/frontend-components-remediations": "^3.2.11",
+    "@redhat-cloud-services/frontend-components-remediations": "^3.2.12",
     "@redhat-cloud-services/frontend-components-translations": "^3.2.8",
-    "@redhat-cloud-services/frontend-components-utilities": "^4.0.6",
-    "@redhat-cloud-services/host-inventory-client": "^1.2.2",
+    "@redhat-cloud-services/frontend-components-utilities": "^4.0.17",
+    "@redhat-cloud-services/host-inventory-client": "^1.5.2",
     "@scalprum/react-core": "^0.7.1",
     "@unleash/proxy-client-react": "^3.5.0",
-    "axios": "^1.6.8",
+    "axios": "^1.7.7",
     "jest-environment-jsdom": "^29.7.0",
     "p-all": "^5.0.0",
     "query-string": "^6.14.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-lines-ellipsis": "^0.15.3",
     "react-redux": "^7.2.9",
-    "react-router-dom": "^6.11.1",
+    "react-router-dom": "^6.26.1",
     "redux": "^4.2.1",
     "redux-promise-middleware": "^6.1.2"
   },
-  "jest": {
-    "coverageDirectory": "./codecov-jest/",
-    "testEnvironment": "jsdom",
-    "collectCoverage": true,
-    "collectCoverageFrom": [
-      "src/**/*.js",
-      "!src/**/stories/*"
-    ],
-    "setupFiles": [
-      "<rootDir>/config/setupTests.js"
-    ],
-    "roots": [
-      "<rootDir>/src/"
-    ],
-    "moduleNameMapper": {
-      "\\.(css|scss)$": "identity-obj-proxy"
-    },
-    "testPathIgnorePatterns": [
-      "src/SmartComponents/SystemDetail/InventoryDetail.test.js",
-      "src/SmartComponents/PatchSet/PatchSet.test.js"
-    ],
-    "transformIgnorePatterns": [
-      "<rootDir>/node_modules/(?!(@patternfly/react-core/|@patternfly/react-icons/|@redhat-cloud-services|@openshift|lodash-es|@patternfly/react-table|@patternfly/react-tokens|p-all)).*$"
-    ]
-  },
```

Comment:
I just moved this to it's own file and added a single transform. Easier to track changes.